packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -1357,7 +1549,83 @@ test.serial('writeGranuleFromApi() writes a granule to PostgreSQL, DynamoDB, and
   t.true(await esGranulesClient.exists(granuleId));
 });
 
-test.serial('writeGranuleFromApi() writes a granule without an execution to PostgreSQL and DynamoDB.', async (t) => {
+test.serial('writeGranuleFromApi() given a partial granule updates only provided fields', async (t) => {
```

Comment:
Same note as above, if we're writing these tests is cmrLink sufficent?   Should we look at files, etc?

Reply:
My thinking is along the lines of 'Yes, but not here' 😄. 

What these tests are really verifying ****_right now_***** (GIANT ASTERISK) is the null/undefined handling because that's the big logic change here. We want to be sure that we can provide a partial object and it's consistent across datastores. I _think_ the best way to test that is to provide the thinnest granule object possible.

Tests to check e.g. "Do files match if I provide only a `files` object in a PATCH request" are absolutely required but I think that's a different can of worms than dealing with unspecified fields.

Reply:
Discussed in huddle/PR review